scripts/appleseedMaya/renderGlobals.py

```diff
@@ -755,115 +774,853 @@ def create(self):
-                        self._addControl(ui=pm.checkBoxGrp(label="Indirect Glossy"), attrName="indirectGlossyAOV")
+                        self._addControl(
+                            ui=pm.checkBoxGrp(
+                                label="Importance Sampling",
+                                columnAttach=(1, "right", 4),
+                                height=18),
+                            attrName="lightImportanceSampling")
+
+                        pm.separator(height=2)
+
+                        with pm.frameLayout("pathTracingFrameLayout", label="Unidirectional Path Tracing",
+                                            collapsable=True, collapse=True):
+                            with pm.columnLayout("pathTracingColumnLayout", adjustableColumn=True, width=subColumnWidth,
+                                                 rowSpacing=2):
+                                pm.separator(height=2)
+
+                                # checkboxes controls
+                                with pm.rowColumnLayout("pathTracingRowColumnLayout", numberOfRows=2,
+                                                        rowOffset=[(1, "top", 2), (2, "both", 2), (3, "both", 2)],
+                                                        rowSpacing=(2, 2)):
+                                    self._addControl(
+                                        ui=pm.checkBoxGrp(
+                                            label="Caustics",
+                                            columnAttach=(1, "right", 4)),
+                                        attrName="caustics")
+
+                                    self._addControl(
+                                        ui=pm.checkBoxGrp(
+                                            label="Direct Lighting",
+                                            columnAttach=(1, "right", 4),
+                                            changeCommand=self.__enableDirectLightingChanged),
+                                        attrName="enableDirectLighting")
+
+                                    self._addControl(
+                                        ui=pm.checkBoxGrp(
+                                            label="Image-Based Lighting",
+                                            columnAttach=(1, "right", 4),
+                                            changeCommand=self.__enableIBLChanged),
+                                        attrName="enableIBL")
+
+                                    self._addControl(
+                                        ui=pm.checkBoxGrp(
+                                            label="Limit Bounces",
+                                            columnAttach=(1, "right", 4),
+                                            changeCommand=self.__limitBouncesChanged),
+                                        attrName="limitBounces")
+
+                                # bounces sliders
```

Comment:
Full line comments should start with a capital and end with a period.

Reply:
Removed, they outlived their usefulness.